### ISSUE TITLE
Typo in NestHydrationJS.js#L282

### ISSUE THREAD

**Issue by jurko-gospodnetic** [CoursePark/NestHydrationJS#3]
The `if` condition in `NestHydrationJS.js` [line 282](https://github.com/CoursePark/NestHydrationJS/blob/master/NestHydrationJS.js#L282) reads:

```
if (columnList.legnth === 0) {
	throw 'invalid structPropToColumnMap format';
}
```

Note the `legnth` --> `length` typo in the used property name, causing this condition to never be valid.

We might also need a test making sure this error handling code works.

**Comment by dvalentiate** [comment]
Good find and ya it is about time to get code coverage on this because clearly it is needed if this wasn't detected earlier.

**Comment by dvalentiate** [comment]
The immediate concern has been addressed with 2a3f075dbc4 but the full code coverage will need to wait.
